src/pages/basic_data/app_admin/central_import/CentralImportCandidateMerge.tsx

```diff
@@ -109,47 +134,118 @@ export const CentralImportCandidateMerge = () => {
           <Typography variant="h1">{t('basic_data.central_import.merge_candidate.result_in_nva')}</Typography>
 
           <CompareFields
-            label={t('basic_data.central_import.merge_candidate.result_id')}
+            candidateLabel={t('basic_data.central_import.merge_candidate.result_id')}
             variant="standard"
-            candidateValue=""
-            registrationValue={values.identifier}
+            candidateValue={candidateIdentifier}
+            registrationValue={registration.identifier}
           />
 
           <CompareFields
-            label={t('common.doi')}
+            candidateLabel={t('common.doi')}
             variant="standard"
             candidateValue={importCandidate.doi || importCandidate.entityDescription?.reference?.doi}
-            registrationValue={values.doi || values.entityDescription?.reference?.doi}
+            registrationValue={registration.doi || registration.entityDescription?.reference?.doi}
+          />
+
+          <CompareFields
+            candidateLabel={t('common.category')}
+            variant="standard"
+            candidateValue={t(
+              `registration.publication_types.${
+                importCandidate.entityDescription?.reference?.publicationInstance.type as PublicationInstanceType
+              }`
+            )}
+            registrationValue={t(
+              `registration.publication_types.${
+                registration.entityDescription?.reference?.publicationInstance.type as PublicationInstanceType
+              }`
+            )}
           />
 
           <CompareFields
-            label={t('registration.description.date_published')}
+            candidateLabel={t('registration.description.date_published')}
             variant="standard"
             candidateValue={displayDate(importCandidate.entityDescription?.publicationDate)}
-            registrationValue={displayDate(values.entityDescription?.publicationDate)}
+            registrationValue={displayDate(registration.entityDescription?.publicationDate)}
+          />
+
+          <CompareFields
```

Comment:
denne ble relativt kompleks og tunglest, men ser ikke noen enklere måte å gjøre det på 🤔 Blir ev. å flytte dette til en egen fil da. Da får man gjenbrukt noe av arraysjekkene, men det blir fortsatt mye av det samme bare at man må sjekke to filer